test/test_build.py

```diff
@@ -216,8 +221,13 @@ def test_it_does_not_run_otherwise(self, mock_get_logger, mock_run):
 
 @patch('steps.build.run')
 @patch('steps.build.get_logger')
+@patch('steps.build.is_supported_ruby_version')
 class TestSetupRuby():
-    def test_no_ruby_version_file(self, mock_get_logger, mock_run, patch_clone_dir):
+    def test_no_ruby_version_file(self, mock_is_supported_ruby_version,
+                                  mock_get_logger, mock_run, patch_clone_dir):
+
+        mock_is_supported_ruby_version.return_value = True
```

Comment:
I understand the result of the truthiness check will be the same, but the implementation actually returns a `1` or `0` not `True` for `False`, so for consistency, we might want this to reflect that.